lib/srv/discovery/fetchers/azure-sync/roleassignments.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package azure_sync
+
+import (
+	"context"
+	"fmt" //nolint:unused // used in a dependent PR
```

Comment:
```suggestion
```

Reply:
Oops, thought the linter was complaining about a second order effect of an unused function. But it was just the import order. I have a wild imagination.

Reply:
Ah yeah, the linter does complain about consts in unused functions, but not about imports in unused functions.